pdf/creator/creator_test.go

```diff
@@ -2415,37 +2415,40 @@ func TestCombineDuplicateDirectObjects(t *testing.T) {
 			"mollit anim id est laborum.")
 		p.SetTextAlignment(TextAlignmentJustify)
 		p.SetMargins(0, 0, 5, 0)
-		for j := 0; j < 7; j++ {
+
```

Comment:
Any reason for the changes in this file?  Isn't it fine to make more objects here for testing the compression?
Also wondering about the page metadata at the bottom, is that relevant to the optimization?

Reply:
I think the file about 1720 KB is not required for this test. After the changes, it has 235 KB.
Add metadata is the easy way to create duplicated direct objects, as I think.